starknet_devnet/starknet_wrapper.py

```diff
@@ -310,11 +321,22 @@ async def deploy(self, deploy_transaction: Deploy) -> Tuple[int, int]:
         try:
             preserved_block_info = self.__update_block_number()
 
-            contract = await self.starknet.deploy(
-                contract_class=contract_class,
-                constructor_calldata=deploy_transaction.constructor_calldata,
-                contract_address_salt=deploy_transaction.contract_address_salt,
-            )
+            if self.config.lite_mode:
```

Comment:
Instead of having to branch here into an `if` and `else`, can't `self.starknet` be earlier set to `LiteStarknet`?

Reply:
In the current state, it will be just moving one ```if``` from here to the constructor because in the second place we have ```InternalDeploy``` and ```LiteInternalDeploy``` but we could look into it when this situation will change in the future.